src/widgets/TextWidget/index.js

```diff
@@ -194,7 +197,9 @@ class TextWidget extends React.Component {
                     textStyle,
                   ]}
                 >
-                  {value !== null && value !== undefined ? value : ''}
+                  {
+                    (value !== null && value !== undefined) ? (customName ? customName(value) : value) : '' // eslint-disable-line
```

Comment:
If `textParser` is provided, the whole text logic should be within the provided textParser.

Instead, you should use `defaultProps` and defined the default textParser as:
```
textParser: value => (value !== null && value !== undefined ? value : ''),
```

Reply:
Done https://github.com/CareLuLu/react-native-web-jsonschema-form/pull/97/commits/659f94fab24a2ee57c5c50e5929d733b0309c88b